src/libraries/System.Text.Json/tests/Serialization/ReadValueTests.cs

```diff
@@ -568,6 +650,94 @@ public static void TooLittleJsonForIntArray(string json)
 
             Assert.Equal(0, reader.BytesConsumed);
         }
+
+        // From https://github.com/dotnet/runtime/issues/882
```

Comment:
nit: do we need the issue number here? I remember @steveharter saying they aren't useful once the issue is fixed (more useful in source).

Reply:
We have such comments elsewhere and I don't see much harm of having a pointer.